apps/site/test/site_web/controllers/schedule/line_test.exs

```diff
@@ -287,447 +282,6 @@ defmodule SiteWeb.ScheduleController.LineTest do
     end
   end
 
-  describe "build_stop_list/2 for Green Line" do
```

Comment:
Are we just deleting these temporarily? If so and if we'd like to add them back in the future I'd recommend opening a ticket and adding a link to it here as a TODO so we don't forget these

Reply:
Or if it's temporary, we can add a `skip` tag here.

Reply:
These haven't actually been deleted, just moved to [diagram_helpers_test.exs](https://github.com/mbta/dotcom/pull/1388/files#diff-f5d4adc72b585b9cffdd568bb393d3890807039426729bd4b4c7c5382aaddd17R1012) (which matches the location of the implementation files.

Reply:
Ah I see, GitHub decided that part of the diff should be collapsed so I missed them. Thanks!